ROADMAP.md

```diff
@@ -13,89 +13,62 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
+  * Enable automated repairs [#2531](https://github.com/rook/rook/issues/2531)
+  * Handle loss of persistent local data [#2533](https://github.com/rook/rook/issues/2533)
   * Minor version upgrades [#2532](https://github.com/rook/rook/issues/2532)
 * Ceph
   * Add support for Ceph Nautilus
-  * Beta release of Ceph-CSI plug-in [#1385](https://github.com/rook/rook/issues/1385)
-    * Support mon failover (feature parity w/ flexvolume) [#2649](https://github.com/rook/rook/issues/2649)
+  * Beta release of CSI plug-in [#1385](https://github.com/rook/rook/issues/1385)
     * Dynamic volume resizing [#1169](https://github.com/rook/rook/issues/1169)
-  * Improved data placement and pool configuration (CRUSH maps) [#560](https://github.com/rook/rook/issues/560)
-  * OSDs
-    * Detect new nodes for OSD configuration [#2208](https://github.com/rook/rook/issues/2208)
-    * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
-    * Allow multiple osds on fast device types (NVMe) [#2640](https://github.com/rook/rook/issues/2640)
-  * File
-    * NFS Ganesha CRD [#1799](https://github.com/rook/rook/issues/1799)
-  * Mgr and plugins
-    * Placement group balancer support (enable the mgr module) [#560](https://github.com/rook/rook/issues/560)
+  * Automatically detect new nodes [#2208](https://github.com/rook/rook/issues/2208)
+  * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
+  * Create NFS Ganesha CRD [#1799](https://github.com/rook/rook/issues/1799)
   * Document metadata backup and disaster recovery [#592](https://github.com/rook/rook/issues/592)
 * CockroachDB
   * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
 * EdgeFS
   * Declare EdgeFS CRDs to be Beta v1 [#2506](https://github.com/rook/rook/issues/2506)
   * Automatic host validation [#2409](https://github.com/rook/rook/issues/2409)
-  * Target VDEVs
-    * Support for flexible write I/O “sync” option [#2367](https://github.com/rook/rook/issues/2367)
+  * Allow for more performance vs. availability tradeoff configuration [#2367](https://github.com/rook/rook/issues/2367)
   * Object
     * OpenStack/SWIFT CRD [#2509](https://github.com/rook/rook/issues/2509)
     * Support for S3 bucket as DNS subdomain [#2510](https://github.com/rook/rook/issues/2510)
-  * Block (iSCSI)
-    * Support Block CSI [#2507](https://github.com/rook/rook/issues/2507)
-  * Mgr
-    * Support for Prometheus Dashboard and REST APIs [#2401](https://github.com/rook/rook/issues/2401)
-    * Support for Management GUI with automated CRD wizards [#2508](https://github.com/rook/rook/issues/2508)
+  * Support Block (iSCSI) CSI [#2507](https://github.com/rook/rook/issues/2507)
+  * Support for Prometheus Dashboard and REST APIs [#2401](https://github.com/rook/rook/issues/2401)
+  * Support for Management GUI with automated CRD wizards [#2508](https://github.com/rook/rook/issues/2508)
```

Comment:
Please add these two:

* Failure domains and zoning provisioning support [#2513](https://github.com/rook/rook/issues/2513)
* Multi-Namespace clusters support [#2878](https://github.com/rook/rook/issues/2878)
* Embedded mode support [#2810](https://github.com/rook/rook/issues/2810)